pages/contact.vue

```diff
@@ -1,15 +1,23 @@
 <template>
   <v-layout justify-center column align-content-center>
     <h1>{{ $t('contact.title') }}</h1>
-    <i18n class="newlines" path="contact.blurb.template" tag="p">
+    <i18n path="contact.blurb.body-faq" tag="p">
       <template v-slot:faq>
-        <nuxt-link :to="localePath('about-faq')" v-text="$t('contact.blurb.faq')" />
+        <nuxt-link :to="localePath('about-faq')">
+          {{ $t('common.faq') }}
+        </nuxt-link>
       </template>
+    </i18n>
+    <i18n path="contact.blurb.body-contact" tag="p">
       <template v-slot:woudc>
-        <b>WOUDC</b>
+        <strong>WOUDC</strong>
```

Comment:
missing `$t()` around "WOUDC" text ?

Reply:
WOUDC is the same in both languages so it was meant to be plain, un-translated text. I guess you may as well put it in the translations for future-proofing though.